SPECS/xapi.spec

```diff
@@ -438,6 +439,10 @@ Coverage files from unit tests
 %endif
 
 %changelog
+* Fri Apr 24 2020 Benjamin Reis <benjamin.reis@vates.fr> - 1.160.2-1.3
+- Patch xapi-1.160.2-open-openflow-port.XCP-ng.patch modified
```

Comment:
Patch added, not modified?

Reply:
right, i might have made the same mistake on 8.1...

Reply:
just pushed a fix. I'll create a PR fixing 8.1